app/assets/stylesheets/components/_post.scss

```diff
@@ -80,6 +83,64 @@ $Post-inlineCodeBgColor: rgba(0, 0, 0, 0.04);
     }
   }
 
+  .Table {
```

Comment:
Can I assume that line 27 breaking SUIT is valid since it reduces complexity? One could also argue that A `.PostTable` component is supposed to be namespaced inside the post. Alternatively, instead of turning it into a different component, one could use a descendent as @pfac suggested, `Post-table`.